File: Csharp/RETANGULO.cs

using System;
using System.Globalization;
internal class Program {
    private static void Main(string[] args) {

        CultureInfo CI = CultureInfo.InvariantCulture;

        double bas, alt;
        Console.Write("Base do retangulo: ");
        bas = double.Parse(Console.ReadLine());
        Console.Write("Altura do retangulo: ");
        alt = double.Parse(Console.ReadLine());

        double area = bas * alt;
        double peri = 2 * (bas + alt);
        double diag = Math.Sqrt(bas * bas + alt * alt);

        Console.WriteLine("AREA = " + area.ToString("F4", CI));
        Console.WriteLine("PERIMETRO = " + peri.ToString("F4", CI));
        Console.WriteLine("DIAGONAL = " + diag.ToString("F4", CI));
    }
}

File: PYTHON/RETANGULO.py
import math

base = float(input("Digite a base do retangulo: "))
alt = float(input("Digite a altura do retangulo: "))

area = base * alt
peri = 2 * base + 2 * alt
diag = math.sqrt(base ** 2 + alt ** 2)

print(f"AREA = {area:.4f}")
print(f"PERIMETRO = {peri:.4f}")
print(f"DIAGONAL = {diag:.4f}")
